src/ansys/dpf/core/operators/build.py

```diff
@@ -182,13 +182,30 @@ def build_operators():
 
     succeeded = 0
     done = 0
+    hidden = 0
+    # List of hidden operators to still expose for retro-compatibility
+    # until they are fully deprecated
+    hidden_to_expose = [  # Use internal names
```

Comment:
@cbellot000 here is the list of exceptions for hidden operators to still expose as a Python module